demo-shell/src/app/components/files/files.component.scss

```diff
@@ -163,9 +167,9 @@
     }
 
     @media (max-device-width: 1024px) {
-        adf-document-list .adf-data-table {
+        adf-document-list .adf-datatable-list {
```

Comment:
why using tag name instead of class name here?

Reply:
You're right!